css/src/atomics/typography.scss

```diff
@@ -0,0 +1,112 @@
+// Heading sizes
+.heading-h1 {
+	font-size: $font-size-1 !important;
+}
+
+.heading-h2 {
+	font-size: $font-size-2 !important;
+}
+
+.heading-h3 {
+	font-size: $font-size-3 !important;
+}
+
+.heading-h4 {
+	font-size: $font-size-4 !important;
+}
+
+.heading-h5 {
+	font-size: $font-size-5 !important;
+}
+
+.heading-h6 {
+	font-size: $font-size-6 !important;
+}
+
+// Non-heading sizes
+.font-size-xxl {
+	font-size: $font-size-0 !important;
+}
+
+.font-size-xl {
+	font-size: $font-size-1 !important;
+}
+
+.font-size-l {
+	font-size: $font-size-2 !important;
+}
+
+.font-size-m {
+	font-size: $font-size-3 !important;
+}
+
+.font-size-s {
+	font-size: $font-size-5 !important;
+}
+
+.font-size-xs {
+	font-size: $font-size-7 !important;
```

Comment:
seems like we're missing some values while including others. how did we come around to deciding which were included and which weren't?

Reply:
So `font-size-1` to `font-size-6` is already included as part of the heading classes . the only one left are `font-size-0` ,`font-size-7`, `font-size-8` and `font-size-9` . out of which i dont think we need a class for `font-size-7:1rem` . So while creating this i made sure to alternatively give class to the one which are already included as a part of heading sizes . 
$font-size-9: 0.75rem !default; // 12px XS
$font-size-8: 0.875rem !default; // 14px SMALL
$font-size-7: 1rem !default; // 16px 
$font-size-6: 1.125rem !default; // 18px MEDIUM
$font-size-5: 1.25rem !default; // 20px 
$font-size-4: 1.5rem !default; // 24px LARGE
$font-size-3: 1.75rem !default; // 28px 
$font-size-2: 2.125rem !default; // 34px XL
$font-size-1: 2.5rem !default; // 40px 
$font-size-0: 3.375rem !default; // 54px XXL

Or the other option is to include all the font-size but then we will have 10 classes and it can go tilll 4XL . Let me know what you think about this .

Reply:
Here's what I'd like to suggestion for now

```
$font-size-9: 0.75rem !default; // 12px XS
$font-size-8: 0.875rem !default; // 14px S
$font-size-7: 1rem !default; // 16px M
$font-size-6: 1.125rem !default; // 18px L
$font-size-5: 1.25rem !default; // 20px XL

$font-size-0: 3.375rem !default; // h0 - let's add this to the heading atomics above.
```

We cover most larger sizes within the `h*` ramp. The `h*` sizes encourage (but don't require) that developers maintain some visual hierarchy with their headings.

The smaller sizes (here listed as `xs - xl`) should be used for other cases (i.e. not headings). For this reason I don't think we need to extend those classes to cover all the values in the typo ramp (however, you'll note that we _could_ do this if for some reason this decision was a mistake).

However, that does leave one value (`$font-size-0`) out in the cold. Since the primary use case for this one is _also a heading_, I think it makes sense to add it to the heading-related atomics instead of the primary oft-used tshirt atomics.